README.md

```diff
@@ -20,7 +19,7 @@ Especially in a CI/CD environment, we need a clear, actionable pass/fail result
 
 ​:running: [Running tasks at the beginning of a deploy](#running-tasks-at-the-beginning-of-a-deploy) using bare pods (example use case: Rails migrations)
 
-This repo also includes related tools for [running tasks](#kubernetes-run) and [restarting deployments](#kubernetes-restart).
+This repo also includes related tools for [running tasks](#krane run) and [restarting deployments](#krane restart).
```

Comment:
We don't mention `krane render` in this section, even though you could argue it is going to be more important than it was in the `kubernetes-deploy` days, since otherwise there's no way to take advantage of the ERB templating

Reply:
+1 definitely need to add a mention of `render` somewhere. Part of that should briefly mention how it can be used with deploy:
`krane render | krane deploy --stdin` sorta deal.